twisted/protocols/policies.py

```diff
@@ -651,7 +651,8 @@ def __init__(self, wrappedFactory, logfilePrefix, lengthLimit=None):
 
 
     def open(self, name):
-        return file(name, 'w')
+        import builtins
+        return builtins.open(name, 'w')
```

Comment:
It doesn't seem to be required.  This is an example which works:
```
class A:
    def open(self, dummy):
        open(dummy, "w")

a = A()
a.open("nada")
```

This example will recurse infinitely:
```
def open(one, two):
    open(one, two)

open("nada", "w")
```
I opted for maximum clarity and safety buy putting builtins in the patch.